src/main/java/com/onepagecrm/models/User.java

```diff
@@ -456,4 +457,13 @@ public User setCountryCode(String countryCode) {
         this.countryCode = countryCode;
         return this;
     }
+
+    public Boolean isNewUser() {
```

Comment:
@lorobode In this scenario there are two options:

1) Use primative `boolean` type.
-> Have only getter and setter

2) If using `Boolean` type:
-> Getter (`#getNewUser`) returns `Boolean`... and therefore possibly `null`
-> Additional helper method `#isNewUser` would return `boolean`... would be `return newUser != null && newUser;`